### DISCUSSION TITLE
Realtime plot

### DISCUSSION THREAD

**Discussion by matiashr** [Blake-Madden/Wisteria-Dataviz#38] [General]
Is it possible to update the graph of a plot ?
Ive used a timer and changed the data of a lineplot, however I cannot find a way to replot ?
resizing the window updates the plot..

**Comment by Blake-Madden** [comment]
Call `Canvas::CalcAllSizes()` to recalculate everything and update the graphs. For example:

```
wxGCDC gdc(parentFrameThatContainsCanvas);
myCanvas->CalcAllSizes(gdc);
```

**Comment by Blake-Madden** [comment]
I've updated all the `SetData()` comments to mention calling `Canvas::CalcAllSizes()` to re-plot the graph. Thanks.
